File: buzz.cpp
#include<iostream>
using namespace std;
bool isBuzz(int n)
{
	if(n%7 == 0 || n%10 == 7)
	{
		return true;
	}
	return false;
}
int main()
{
	int n;
	cout<<"Enter a number : ";
	cin>>n;
	if(isBuzz(n))
	{
		cout<<"Is a Buzz number."<<endl;
	}
	else
	{
		cout<<"Not a Buzz number."<<endl;
	}
}

File: buzz.py
def isBuzz(n:int)->bool:
    if n%7 == 0 or n%10 == 7:
        return True
    return False

if __name__ == "__main__":
    n = int(input("Enter a number : "))
    if isBuzz(n):
        print("Is a Buzz number.")
    else:
        print("Not a Buzz number.")
    